### PR TITLE
implementation of the start analysis button on frontend

### PR DESCRIPTION
**Pull request by Arielpetit**
Done by @Arielpetit  and @Calebasah

### CODE REVIEW

**Comment by Arielpetit** [comment]
hello sir please review our ticket

File: templates/results.html
Comment (stephane-segning) [inline templates/results.html:1]: Make this match with the `base.html` file, so that no code repeat

**Review by stephane-segning — CHANGES_REQUESTED** [review]
Very goo dcode, @Arielpetit & @Calebasah. Review these points, please:

File: templates/results.html
Comment (Calebasah) [reply templates/results.html:1]: > Make this match with the `base.html` file, so that no code repeat

Sir the update was done as suggested

Comment (stephane-segning) [reply templates/results.html:1]: Good job @Calebasah. Now add the script link to the `base.html`

File: app.py
Comment (stephane-segning) [inline app.py:1]: Why are you changing the port?

**Review by stephane-segning — CHANGES_REQUESTED** [review]

File: app.py
Comment (Calebasah) [reply app.py:1]: The port was not responding, but the port has been updated

**Review by stephane-segning — APPROVED** [review]

**Pull request merged**